programs/drift/src/instructions/user.rs

```diff
@@ -2360,6 +2367,14 @@ pub fn handle_begin_swap(
                 ErrorCode::InvalidSwap,
                 "only allowed to pass in ixs to token or openbook or Jupiter v3 or v4 programs"
```

Comment:
is this is msg out of date? (doesnt include marinade program?)

Reply:
yeah